pkg/tuned/run.go

```diff
@@ -16,26 +16,31 @@ package tuned
 
 import (
 	"bufio"
+	"context"
 	"fmt"
 	"os/exec"
 	"strings"
+	"time"
 
 	"k8s.io/klog/v2"
 )
 
-func TunedCreateCmd(debug bool) *exec.Cmd {
+func TunedCreateCmdline(debug bool) (string, []string) {
 	args := []string{"--no-dbus"}
 	if debug {
 		args = append(args, "--debug")
 	}
-	return exec.Command("/usr/sbin/tuned", args...)
+	return "/usr/sbin/tuned", args
+}
+
+func TunedCreateCmd(command string, args []string) *exec.Cmd {
+	return exec.Command(command, args...)
 }
```

Comment:
I'd go as far as remove this function, is now so shallow it's value is debatable